gob/k8s/README.md

```diff
@@ -329,8 +328,9 @@ to the old version.
 You can simulate this with:
 
 ```
-:; kubectl delete svc/gen-growthhack
+:; kubectl delete -f k8s/gen-growthhack
 service "gen-growthhack" deleted
+replicationcontroller "gen-growthhack" deleted
```

Comment:
I'd actually leave this command as it was previously written.  Here we're trying to demonstrate that reversions can be handled by changes in service discovery.  Deleting the service is enough to remove the instance from service discovery, and doesn't require tearing down the instance itself.